easybuild/easyconfigs/g/gnuplot/gnuplot-5.2.2-foss-2018a.eb

```diff
@@ -0,0 +1,46 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg/Luxembourg Centre for Systems Biomedicine
+# Authors::   Fotis Georgatos <fotis.georgatos@uni.lu>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-97.html
+##
+easyblock = 'ConfigureMake'
+
+name = 'gnuplot'
+version = '5.2.2'
+
+homepage = 'http://gnuplot.sourceforge.net/'
+description = """Portable interactive, function plotting utility"""
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = [('https://sourceforge.net/projects/gnuplot/files/gnuplot/%(version)s', 'download')]
+sources = [SOURCE_TAR_GZ]
+checksums = ['a416d22f02bdf3873ef82c5eb7f8e94146795811ef808e12b035ada88ef7b1a1']
+
+dependencies = [
+    ('cairo', '1.14.12'),
+    ('libjpeg-turbo', '1.5.3'),
+    ('libpng', '1.6.34'),
+    ('libgd', '2.2.5'),
```

Comment:
@reedts Easyconfig file for this `libgd` dep is still missing?

Reply:
@boegel Should be included now.

Reply:
looks good now indeed :+1: